crates/ruff_linter/src/codes.rs

```diff
@@ -383,6 +383,7 @@ pub fn code_to_rule(linter: Linter, code: &str) -> Option<(RuleGroup, Rule)> {
         (Flake8Comprehensions, "18") => (RuleGroup::Stable, rules::flake8_comprehensions::rules::UnnecessaryLiteralWithinDictCall),
         (Flake8Comprehensions, "19") => (RuleGroup::Stable, rules::flake8_comprehensions::rules::UnnecessaryComprehensionInCall),
         (Flake8Comprehensions, "20") => (RuleGroup::Preview, rules::flake8_comprehensions::rules::UnnecessaryDictComprehensionForIterable),
+        (Flake8Comprehensions, "21") => (RuleGroup::Preview, rules::flake8_comprehensions::rules::UnnecessaryLiteralWithinDequeCall),
```

Comment:
We should either open an upstream issue and see if the rule gets accepted and implemented or implement the Rule as a RUFF rule. We otherwise risk that we end up with incompatible rule codes between upstream and ruff (if flake8-comprehension adds a new rule that's not `UnnecessaryLiteralWithinDequeueCall`)